### DISCUSSION TITLE
More Maintainers Needed?

### DISCUSSION THREAD

**Discussion by grablair** [hammem/monarchmoney#56] [General]
It seems we've gotten a bit blocked on approvals on the recent PRs. I know that this is open source and maintainers have lives to live, and that is 100% okay!  Maybe we need a few more maintainers to keep the project growing?

I'm happy to become a maintainer myself.

CC: @hammem @CalvinChanCan

**Comment by hammem** [comment]
Completely in favor!  Unless @grablair and @CalvinChanCan object, I'll add you both.

**Reply by grablair** [reply]
Sounds good to me! Feel free to add me!

**Reply by hammem** [reply]
@grablair , you're added!
